### PR TITLE
PT-3350: Add ability to enter groups in the owner and collaborator fields of the filtering panel

### PR DESCRIPTION
**Pull request by veronikaslc**

### CODE REVIEW

**Comment by sashaandjic** [comment]
Awesome! Just one small thing to adjust. Either widen the collaborator field or shorten the diplayed text.
Please see the attached screenshot.
screen shot 2017-12-21 at 12 30 36 pm

**Comment by sashaandjic** [comment]
Can we just say "Type user or group" in both fields?
pt-3350

File: components/users/api/pom.xml [hunk 1]
```diff
@@ -29,7 +29,7 @@
   <name>PhenoTips - Users and Groups - Java APIs</name>
 
   <properties>
-    <coverage.instructionRatio>0.93</coverage.instructionRatio>
+    <coverage.instructionRatio>0.86</coverage.instructionRatio>
   </properties>
```
Comment (marta-) [inline components/users/api/pom.xml]: New rule: no more test coverage reduction. All new code must come with tests.

**Comment by sashaandjic** [comment]
I can no longer modify the owner.

**Comment by a-gvo** [comment]
I have the same issue as @sashaandjic. Tried Chrome and Firefox on mac. I can change the owner if i type the full username. If I select one of the dropdown options though, the input clears.

**Comment by veronikaslc** [comment]
@sashaandjic @a-gvo Did a little refactoring & improvements.

File: components/widgets/ui/src/main/resources/PhenoTips/Widgets.xml [hunk 2]
```diff
@@ -5143,17 +5143,12 @@ document.observe("xwiki:dom:updated", init);
   var init = function(event) {
     var containers = (event &amp;&amp; event.memo.elements) || [$('body')];
     containers.each(function(container) {
-      container.select('input.suggestWorkgroups').each(function(input) {
```
[the remark this answers is not in the document]
Comment (veronikaslc) [reply components/widgets/ui/src/main/resources/PhenoTips/Widgets.xml]: @sdumitriu pushed a commit (hope, I understood right that you want a separate suggest workgrop widget)

**Pull request merged**